academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -21,6 +21,11 @@ ms.date: 0
 
 ### Create indexing resources
 
+```CMD
+
+kesm.exe CreateIndexResources --IndexResourceName "alchmakesindexres" --MakesPackage "https://alchmakesstore.blob.core.windows.net/makes/2020-01-30/" --MakesIndexResourceConfigFilePath "makesIndexResConfig" --Region westus2 --AzureCredentialsFilePath "my.azureauth"
```

Comment:
>kesm.exe CreateIndexResources --IndexResour [](start = 0, length = 43)

I have removed this from the v2-launch (in links) and have another branch w/ all of this + sub graphing.